MdeModulePkg/Bus/Pci/XhciDxe/XhciSched.c

```diff
@@ -1736,8 +1746,6 @@ XhcMonitorAsyncRequests (
     if (ProcBuf != NULL) {
       gBS->FreePool (ProcBuf);
     }
-

```

Comment:
While changing here, can you fix the grammar in the comment above RestoreTpl in line 1738?